packages/application-shell/src/components/custom-view-flop-flip-provider/custom-view-flop-flip-provider.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode, useMemo } from 'react';
+import memoryAdapter from '@flopflip/memory-adapter';
+import { ConfigureFlopFlip } from '@flopflip/react-broadcast';
+import { TFlags } from '@flopflip/types';
+import { TFetchLoggedInUserQuery } from '../../types/generated/mc';
+
+type TCustomViewFlopFlipProviderProps = {
+  projectKey?: string;
+  user?: TFetchLoggedInUserQuery['user'];
+  flags?: TFlags;
+  children: ReactNode;
+};
+
+function CustomViewFlopFlipProvider(props: TCustomViewFlopFlipProviderProps) {
+  const adapterArgs = useMemo(
+    () => ({
+      user: {
```

Comment:
`user` is required in the adapter args.
We don't actually need anything here because they're use to evaluate the flags, but that's already done in the host application. However, I just wanted to add some basic information we already have available.